js/index.html

```diff
@@ -57,24 +57,9 @@ <h2>Installation</h2><h3><a href="http://npmjs.com">NPM</a></h3><p>A dependency
 <h2>Code style &amp; Conventions</h2><ul>
 <li>https://github.com/google/eslint-config-google</li>
 </ul>
-<h2>License</h2><p>Copyright (c) 2018 Nexmo, Inc. All rights reserved. Licensed only under the Nexmo Stitch SDK License Agreement (the &quot;License&quot;) located at</p>
-<p>By downloading or otherwise using our software or services, you acknowledge that you have read, understand and agree to be bound by the Nexmo Stitch SDK License Agreement and Privacy Policy.</p>
+<h2>License</h2><p>Copyright (c) 2018 Nexmo, Inc. All rights reserved. Licensed only under the Nexmo Client SDK License Agreement (the &quot;License&quot;) located at</p>
+<p>By downloading or otherwise using our software or services, you acknowledge that you have read, understand and agree to be bound by the Nexmo Client SDK License Agreement and Privacy Policy.</p>
 <p>You may not use, exercise any rights with respect to or exploit this SDK, or any modifications or derivative works thereof, except in accordance with the License.</p>
-<h2>Authors</h2><ul>
```

Comment:
@kapekost any reason we removed authors here?

Reply:
getting a bit overwhelming to maintain the list... we can see collaborators in the main repo and update if needed... 🤷🏻‍♂️